src/Network/NWPathMonitor.cs

```diff
@@ -33,6 +41,12 @@ public NWPathMonitor ()
 		public NWPathMonitor (NWInterfaceType interfaceType)
 		{
 			InitializeHandle (nw_path_monitor_create_with_type (interfaceType));
+			InternalSetUpdatedSnapshotHandler ((path) => {
+				currentPath = path;
+				if (userSnapshotHandler != null) {
+					userSnapshotHandler(currentPath);
```

Comment:
same (style)

also this might be better added in a separate method (DRY - don't repeat yourself)

Reply:
Yep, I have the same point of view.